Mathlib/Algebra/Algebra/Subalgebra/Basic.lean

```diff
@@ -1509,4 +1509,5 @@ theorem mem_subalgebraOfSubring {x : R} {S : Subring R} : x ∈ subalgebraOfSubr
   Iff.rfl
 #align mem_subalgebra_of_subring mem_subalgebraOfSubring
 
+attribute [nolint simpNF] AlgEquiv.subalgebraMap_apply_coe AlgEquiv.subalgebraMap_symm_apply_coe
```

Comment:
It's not at all clear to me why these lemmas are problematic; is their statement changing?

Reply:
No, `simp` has changed!

Reply:
Well, some of these lemmas are generated by running simp and looking at the output; so the two aren't mutually exclusive.